opta/module_processors/base.py

```diff
@@ -89,16 +89,17 @@ def process(self, module_idx: int) -> None:
         if isinstance(self.module.data.get("public_uri"), str):
             self.module.data["public_uri"] = [self.module.data["public_uri"]]
 
-        new_uris: list[str] = []
-        public_uri: str
-        for public_uri in self.module.data["public_uri"]:
-            if public_uri.startswith("/"):
-                new_uris.append(f"all{public_uri}")
-            elif public_uri.startswith("*"):
-                new_uris.append(f"all{public_uri[1:]}")
-            else:
-                new_uris.append(public_uri)
-        self.module.data["public_uri"] = new_uris
+        if "public_uri" in self.module.data:
```

Comment:
@rabbitfang Can't we just use a getOrDefault sort of functionality and get rid of the conditional statement and use an empty List as a default.